frontend/javascripts/oxalis/view/arbitrary_view.js

```diff
@@ -208,30 +212,6 @@ class ArbitraryView {
     this.needsRerender = true;
   }
 
-  renderToTexture(renderBucketIndices?: boolean = false): Uint8Array {
-    const { renderer, scene } = getSceneController();
-
-    renderer.autoClear = true;
-    let { width, height } = getInputCatcherRect(Store.getState(), ArbitraryViewport);
-    width = Math.round(width);
-    height = Math.round(height);
-
-    const { camera } = this;
-
-    renderer.setViewport(0, 0, width, height);
-    renderer.setScissorTest(false);
-    renderer.setClearColor(0x222222, 1);
```

Comment:
That particular clear color was problably not very important, right?

Reply:
I had a look at the `getRenderedBucketsDebug` method which used `renderToTexture` before and couldn't find that the clear color would be important in any way for that.